src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,27 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
```

Comment:
Consider renaming the method from `getEmployByOrder` to `getEmployeesSortedByAgeAndName` to more accurately describe its behavior and the order in which employees are sorted.